Uni_life_tracker/app/src/main/java/com/generic/ult/MainActivity.java

```diff
@@ -18,7 +18,7 @@ protected void onCreate(Bundle savedInstanceState) {
     }
     //https://www.youtube.com/watch?v=bgIUdb-7Rqo
    private void openSignup() {
-        Intent intent = new Intent(this, Signup.class);
+        Intent intent = new Intent(this, expenses_landing.class);
```

Comment:
When I tried running the add expense page it just crashed. If you wanna see what the errors are you could "Logcat" at the bottom of the screen and change the Verbose dropdown into error. Here is a video for reference: https://www.youtube.com/watch?v=zUiJ4Qig_Ic

Reply:
It runs for me, I cam push it again and try updating I may have no pushed the latest changes